pkg/controller/secret_control.go

```diff
@@ -30,7 +32,7 @@ import (
 
 // SecretControlInterface manages certificates used by TiDB clusters
 type SecretControlInterface interface {
-	Create(certOpts *TiDBClusterCertOptions, cert []byte, key []byte) error
+	Create(tc *v1alpha1.TidbCluster, certOpts *TiDBClusterCertOptions, cert []byte, key []byte) error
```

Comment:
This API is not generic, I prefer passing owner reference, which abstracts the real owner away